lib/author.rb

```diff
@@ -0,0 +1,35 @@
+require './lib/book'
```

Comment:
This is only needed in the test file since you're only running the files from there.